README.md

```diff
@@ -212,6 +217,9 @@ Yes it is! This plugin comes with .po and .mo files. It is already translated in
 
 ## Changelog ##
 
+### 1.3.1 ###
+* Improvement: Optimized database calls while displaying Five Star rating notice
```

Comment:
I don't think we should add this in the changelog

Reply:
Done